apps/gadgetron/CMakeLists.txt

```diff
@@ -69,7 +70,8 @@ target_link_libraries(gadgetron
         gadgetron_core
         gadgetron_toolbox_log
         boost
-        BLAS)
+        BLAS 
+        ${PUGIXML_LIBRARIES} )
```

Comment:
Again, ${PUGIXML_LIBRARIES} is not required here, and even if it was, we should use a target based approach instead, similar to the rest of the current CMake

Reply:
same here. without this, it has linkage error. But how could I do the target based method?